### DISCUSSION TITLE
Open Vt2 Shell

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1377] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
So yeah @3kh0 or other TN members might not want me posting this here but...
OPEN VT2 SHELL!!!
https://discord.com/channels/419123358698045453/1316548542172631040
All credit goes to people who discovered this.
This is untested and unstable.

1) Download[ tunety.bin](https://omada.cafe/contact/crystals/blog/assets/tunety.bin) and flash it to a USB or SD card using chromeos recovery utility, fedora writer, etcher, rufus, or your another application. Any of them will work fine.
2) Sign in on your chromebook
3) Plug in the USB drive or SD card
4) Wait to be rebooted into a root shell.

If this does not work for you, try at least 25 times before complaining. It involves multiple race conditions, and is very unstable. Once the exploit is ran though, the shell is there until you close it.

To prevent google from finding the method used for this exploit, we are not releasing how to produce this file.

Crystals post: https://omada.cafe/contact/crystals/blog/article/2024/12/11/
https://cdn.discordapp.com/attachments/1316548542172631040/1316548542621417472/tunety.bin?ex=675b72e0&is=675a2160&hm=e4b39e8fb00345708f3a5938248fbe6824caeb10b837eb926b943e60d0625116&)

**Comment by schoolexploitkid** [comment]
wait like im kinda not supposed to release this but pls dont delete

**Reply by Charlieis1nsan3** [reply]
does this work on enrolled Chromebooks? but I'm copying this so if it gets removed I'll have it.

**Reply by schoolexploitkid** [reply]
idk, it didnt work for me but like some people have tested and succeeded.

**Comment by ghost** [comment]
Br o how do u do the safety reset thingy? it was added at 131 chrome os

**Reply by S-PScripts** [reply]
you can't on enrolled chromebooks

 i made a post about it https://github.com/3kh0/ext-remover/discussions/1375

**Comment by sdfndgmuyrj6e56eyt** [comment]
post the one that allows you to downgrade without a usb
